compiler/rustc_trait_selection/src/traits/util.rs

```diff
@@ -504,3 +505,21 @@ impl<'tcx> TypeFolder<TyCtxt<'tcx>> for PlaceholderReplacer<'_, 'tcx> {
         }
     }
 }
+
+pub fn sizedness_fast_path<'tcx>(tcx: TyCtxt<'tcx>, predicate: ty::Predicate<'tcx>) -> bool {
```

Comment:
Maybe name it `is_sized_pred_for_trivially_sized_type`?

Reply:
hmm... I guess. It's nonobvious enough that you need to read the function anyway unless you already know what it is, then the name is unique enough.